examples/qwik/src/routes/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { component$, useSignal } from '@builder.io/qwik'
+
+type Message = {
+  id: string
+  author: string
+  message: string
+  createdAt: string
+}
+
+export const GetAllMessagesQuery = /* GraphQL */ `
+  query GetAllMessages($first: Int!) @live {
+    messageCollection(first: $first) {
+      edges {
+        node {
+          id
+          author
+          message
+          createdAt
+        }
+      }
+    }
+  }
+`
+
+export default component$(async () => {
+  const messages = useSignal<Message[]>([])
+
+  const {
+    data: { messageCollection }
+  }: {
+    data: { messageCollection: { edges: { node: Message }[] } }
+  } = await fetch(import.meta.env.VITE_GRAFBASE_API_URL as string, {
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json'
```

Comment:
We'll need to pass the `authorization` header here. Does `component$` run on the server?